api/src/main/java/org/apache/cloudstack/api/APICommand.java

```diff
@@ -50,4 +49,5 @@
     RoleType[] authorized() default {};
 
     Class<?>[] entityType() default {};
+
```

Comment:
nit - unnecessary change, you may want to use `git add -p` to check what's being commited.

Reply:
Fixed.

Reply:
nit - still see newline changes in `APICommand`